### ISSUE TITLE
Feature request: opening boxes

### ISSUE THREAD

**Issue by Crysez** [EasyFarm/EasyFarm#245]
Hey, 

Thanks again for the program.  Have been loving it for a few months now. 

Have you considered adding a feature to open boxes (emblazoned reliquaries) & collect items from them, specifically in escha zones?  

There some good stuff in them, including: beads, silt, and ore.

**Comment by Mercurian** [comment]
Just throwing a comment in that I'd be happy to see it to, but it might be tough. It would have to choose between box and aggro, or you get stuck in a loop: box > hit, box > hit, box > hit and die.

And you'd have to run a secondary inventory program to move/drop garbage or have that built in too.

**Comment by Crysez** [comment]
For inventory: just use treasury addon.
For dying: I suppose it could kill things first, then open the box?

**Comment by Mercurian** [comment]
It doesn't always work for priorities though, ever noticed that you'd have aggro, but continue searching for a new mob for a few seconds before it went after the aggro? I assume that would lead to alot of deaths.

**Comment by Mykezero** [comment]
I actually haven't played the retail version of the game in a really long time. Are the boxes complex to open?

**Comment by Crysez** [comment]
No, you literally target the box, then hit enter.  No codes/combinations/games involved.

**Comment by Mercurian** [comment]
That's not entirely true, gold boxes have a game. higher/lower, and reisen boxes don't have a take all

**Comment by Crysez** [comment]
The zitah & ru'aun boxes have a game?  Im certain ive gotten a bunch of ore from those boxes without ever playing a game.

**Comment by Mercurian** [comment]
Blue: no game
Brown: no game
Gold: higher/lower game

**Comment by Crysez** [comment]
Ah, you're right.  I was mixing up gold & brown.  

I think the golds are shitty anyway, though?  Arent they just exp rings?

Allow me to restate my feature request:  open brown boxes onry (since they have the goodies & are easy to open)

**Comment by Mercurian** [comment]
I believe the whole reason a box open never existed was because boxes couldn't be distinguished by color (abyssea)

**Comment by Crysez** [comment]
I will have to check it out over the weekend.  Im betting the boxes have IDs associated with them, and they are probably color-specific.

**Comment by zaxtiss** [comment]
you can tell the differance i know that since an diff bot could MP could
tell what boxes to open

**Comment by Mykezero** [comment]
Yeah, Numpty did a really good job on MyProject with all the different features. I'm pretty sure the EliteAPI would have the functionality to do this.

**Comment by Mercurian** [comment]
Any word on making this work or no?

**Comment by Mykezero** [comment]
I'm not familiar with the Escha; Is it easy to get access to those areas? Sorry, I haven't played retail in a really long time until now.

**Comment by Mercurian** [comment]
You can access the first area of escha from a brand new account in a couple hours, if you still have an account at all you can get there in a couple minutes since you already have game progress/level

**Comment by zaxtiss** [comment]
just to let you know you can go right to the first to escha zones wi9th no
quests/missions

**Comment by Mercurian** [comment]
Want to bump this because I see other bots auto opening boxes

**Comment by zaxtiss** [comment]
Most likely using a private ashita plugin
